src/commands/ReminderCommand/utils/help.js

```diff
@@ -0,0 +1,11 @@
+const showInstructions = (message) => {
+    let helpString =  "The reminder command has the following structure: !schedule (#channelname) <date> <time> <AM / PM> (Timezone)\n" +
```

Comment:
Instead of writing '!schedule', the prefix '!' should come from the .env, since it can be different.
Maybe using a template string.

Reply:
I think you should also explain that the message to be scheduled will be asked after you write the command

Reply:
Since the timezone is optional, maybe mention that by default its europe/lisbon

Reply:
I think the readability could be improved using a MessageEmbed (https://discord.js.org/#/docs/main/stable/class/MessageEmbed)

Something like this: 
![image](https://user-images.githubusercontent.com/32744387/116560597-6fbe5600-a8f9-11eb-8491-cfaf01fab4dc.png)


But it's not that important, I'll leave it up to you